rustsec/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name         = "rustsec"
 description  = "Client library for the RustSec security advisory database"
-version      = "0.30.2"
+version      = "0.30.3"
```

Comment:
I'm assuming gix and tame-index are not public dependencies, is that true?

Reply:
I'm uncertain offhand

Reply:
Okay, looks like the rustsec public API surface is pretty porous and at the very least includes `fs_err` which was semver-incompatibly bumped in #1352, so bumping this to 0.31.